### ISSUE TITLE
t.p.failure.startDebugMode() can't be implemented as log observer

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7571]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#309|
|Type|enhancement|
|Created|2003-10-02 03:35:45Z|

<details><summary>Searchable metadata</summary>

```
trac-id__309 309
type__enhancement enhancement
reporter__itamarst itamarst
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1065065745000000 1065065745000000
changetime__1468100185734743 1468100185734743
version__ 
owner__ 
cc__itamarst
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
This may not be possible due to Failure-cleaning in Deferreds.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
I'm going to assume this is not going to work due to the mentioned issue.
Changed title.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub commented|
|-|-|

In changeset 56e1c4a559639643f9e9366612984da0b5f3f68c
```
#!CommitTicketReference repository="" revision="56e1c4a559639643f9e9366612984da0b5f3f68c"
Merge pull request #309 from twisted/8580-scop-files

Author: scop
Reviewer: rodrigc
Fixes: #8580

More file close related fixes
```
